Lesson4/step2/project.py

```diff
@@ -251,209 +243,207 @@ def gdisconnect():
     h = httplib2.Http()
     result = h.request(url, 'GET')[0]
     if result['status'] == '200':
-        del login_session['access_token']
```

Comment:
Ugly and inflexible.  Any changes to the session will have to be manually updated.  Use an iterator to remove session values.